src/OVAL/probes/SEAP/generic/strbuf.c

```diff
@@ -282,6 +284,7 @@ size_t strbuf_fwrite (FILE *fp, strbuf_t *buf)
 
 ssize_t strbuf_write (strbuf_t *buf, int fd)
 {
+#ifdef HAVE_UIO_H
```

Comment:
I prefer to have functions inside `#ifdef` blocks than vice versa.

Reply:
OK